transports/quic/Cargo.toml

```
@@ -0,0 +1,42 @@
+[package]
+name = "libp2p-quic"
+version = "0.7.0"
+authors = ["David Craven <david@craven.ch>", "Parity Technologies <admin@parity.io>"]
+edition = "2021"
+description = "TLS based QUIC transport implementation for libp2p"
+repository = "https://github.com/libp2p/rust-libp2p"
+license = "MIT"
+
+[dependencies]
+async-std-crate = { package = "async-std", version = "1.12.0", default-features = false, optional = true }
+async-trait = "0.1.50"
+futures = "0.3.15"
+futures-timer = "3.0.2"
+if-watch = "2.0.0"
+libp2p-core = { version = "0.36.0", path = "../../core" }
```

```suggestion
libp2p-core = { version = "0.36.0", path = "../../core", default-features = false }
```

At least until https://github.com/libp2p/rust-libp2p/pull/2918 is merged.

---

#2918 is merged now 🙂